### PR TITLE
40819-add-and-remove-meber

### PR DESCRIPTION
**Pull request by NgengChhengkim**

### CODE REVIEW

File: app/views/rails_admin/main/edit_course.html.erb [hunk 1]
```diff
@@ -51,7 +51,7 @@
     </div>
 
     <div role="tabpanel" class="tab-pane" id="members">
-
+      <%= render "rails_admin/main/add_trainee_to_course"%>
```
Comment (vietanhbkaaa) [inline app/views/rails_admin/main/edit_course.html.erb]: insert space before %>

**Comment by vietanhbkaaa** [comment]
OK

**Comment by VuongLong** [comment]
OK

**Comment by yongsokheng** [comment]
OK

File: lib/rails_admin/add_trainee_to_course.rb [hunk 2]
```diff
@@ -34,17 +34,21 @@ class AddTraineeToCourse < RailsAdmin::Config::Actions::Base
...
             if request.post?
-              course_params = params.require(:course).permit user_ids: []
-              if object.update_attributes course_params
-                flash[:success] = t "admin.actions.updated"
-                redirect_to show_path "course", @course
-              else
-                render "add_trainee_to_course"
-                flash[:danger] = t "admin.actions.not_updated"
+              user_param_ids = params[:course][:user_ids].map(&:to_i).reject{|id| id == 0}
+              user_course_ids = object.users.map(&:id)
```
Comment (hellovietnam93) [inline lib/rails_admin/add_trainee_to_course.rb]: remove ()

File: lib/rails_admin/add_trainee_to_course.rb [hunk 3]
```diff
@@ -34,17 +34,21 @@ class AddTraineeToCourse < RailsAdmin::Config::Actions::Base
...
+              user_add_ids.each do |user_id|
+                object.user_courses.create user_id: user_id, leader_id: current_user.id
+              end
+
+              user_remove_ids.each do |user_id|
+                user_course = UserCourse.find_by course_id: object.id, user_id: user_id
+                user_course.destroy
               end
+              flash[:success] = t "admin.actions.updated"
+              redirect_to "#{edit_course_path(Course, object)}#members"
```
Comment (hellovietnam93) [inline lib/rails_admin/add_trainee_to_course.rb]: dùng cái "course" thay cho Course

**Comment by hellovietnam93** [comment]
OK

**Pull request merged**